src/app/pipes/duration.pipe.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { DurationPipe } from './duration.pipe';
+
+describe('DurationPipe', () => {
+  let pipe: DurationPipe;
+  it('create an instance', () => {
+    pipe = new DurationPipe();
```

Comment:
An instance of the object being tested should be re-initialized in every test. Use `beforeEach`
Additionally, pipes can be tested using Angular: TestBed module with a pipe provider + `TestBed.get()` in each test.